tests/acceptance-tests/src/acceptance/http_closed_test.go

```diff
@@ -0,0 +1,37 @@
+package acceptance_test
+
+import (
+        . "github.com/onsi/ginkgo"
+        . "github.com/onsi/gomega"
+        . "github.com/onsi/gomega/gexec"
+
+        "github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+        "github.com/cloudfoundry-incubator/cf-test-helpers/generator"
+        "os/exec"
+)
+
+var _ = Describe("Http is closed", func() {
+
+        It("for apps", func() {
+                appName := generator.PrefixedRandomName("CATS-APP-")
+                Expect(cf.Cf(
+                        "push", appName,
+                        "--no-start",
+                        "-b", config.StaticFileBuildpackName,
+                        "-p", "../../example_apps/static_app",
+                        "-d", config.AppsDomain,
+                ).Wait(DEFAULT_TIMEOUT)).To(Exit(0))
+                Expect(cf.Cf("start", appName).Wait(CF_PUSH_TIMEOUT)).To(Exit(0))
+
+		uri := "http://" + appName + "." + config.AppsDomain
+        	_, err := exec.Command("curl", "-v", "-s", "--connect-timeout", "11", "--retry", "0", uri).CombinedOutput()
+                Expect(err).To(ContainSubstring("exit status 28"))
```

Comment:
The indentation is off here. Could you run `go fmt` over this file? https://blog.golang.org/go-fmt-your-code